tcs-service/src/main/java/com/transformuk/hee/tis/tcs/service/service/mapper/CurriculumMembershipMapper.java

```diff
@@ -140,30 +125,37 @@ public List<CurriculumMembership> programmeMembershipDtosToCurriculumMemberships
   private ProgrammeMembershipDTO curriculumMembershipToProgrammeMembershipDto(
       CurriculumMembership curriculumMembership) {
     ProgrammeMembershipDTO result = new ProgrammeMembershipDTO();
+    ProgrammeMembership programmeMembership = curriculumMembership.getProgrammeMembership();
 
-    result.setId(curriculumMembership.getId());
-    result.setProgrammeMembershipType(curriculumMembership.getProgrammeMembershipType());
-    result.setProgrammeStartDate(curriculumMembership.getProgrammeStartDate());
-    result.setProgrammeEndDate(curriculumMembership.getProgrammeEndDate());
-    result.setLeavingDestination(curriculumMembership.getLeavingDestination());
-    result.setLeavingReason(curriculumMembership.getLeavingReason());
-    Programme programme = curriculumMembership.getProgramme();
+    result.setId(curriculumMembership.getId()); //use CM ID
+    result.setProgrammeMembershipId(programmeMembership.getId());
+    result.setProgrammeMembershipType(programmeMembership.getProgrammeMembershipType());
+    result.setProgrammeStartDate(programmeMembership.getProgrammeStartDate());
+    result.setProgrammeEndDate(programmeMembership.getProgrammeEndDate());
+    result.setLeavingDestination(null);
```

Comment:
Is this not the same as leaving it unset (null by default)?

Reply:
Yes, I think I included it explicitly to make it clear it was not being not-set by accident 😁 . However - good catch: it should actually be set to the PM's leaving destination, as per James's decision on where that field should reside.